tests/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ add_subdirectory(tools)
 if (OE_SGX)
     add_subdirectory(debugger)
     add_subdirectory(host_verify)
+    add_subdirectory(mbed)
```

Comment:
Does this modification mean to enable tests in `WIN32_SIMULATION` mode? I think this is the only missing configuration before this change (not sure we have that config in CI/CD).

Reply:
I think this flag is left over and should probably be removed. There is no such thing as Windows simulation mode.

Reply:
Then, I think the mbedtls tests are already enabled (based on the checks `ADD_WINDOWS_ENCLAVE_TESTS OR USE_CLANGW`). Can you double check? To avoid confusion, may we can have another PR to remove this flag and re-org the logic in this file.

Reply:
Oh you're right, they do already run. I'll close this. Thanks!